includes/Modules/Subscribe_With_Google.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Subscribe_With_Google
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules;
+
+use Google\Site_Kit\Core\Assets\Asset;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Modules\Module;
+use Google\Site_Kit\Core\Modules\Module_Settings;
+use Google\Site_Kit\Core\Modules\Module_With_Assets;
+use Google\Site_Kit\Core\Modules\Module_With_Assets_Trait;
+use Google\Site_Kit\Core\Modules\Module_With_Settings;
+use Google\Site_Kit\Core\Modules\Module_With_Settings_Trait;
+use Google\Site_Kit\Core\Util\Method_Proxy_Trait;
+use Google\Site_Kit\Modules\Subscribe_With_Google\EditPost;
+use Google\Site_Kit\Modules\Subscribe_With_Google\Header;
+use Google\Site_Kit\Modules\Subscribe_With_Google\InlineEditPost;
+use Google\Site_Kit\Modules\Subscribe_With_Google\ManagePosts;
+use Google\Site_Kit\Modules\Subscribe_With_Google\Settings;
+use Google\Site_Kit\Modules\Subscribe_With_Google\SinglePost;
+
+/**
+ * Class representing the Subscribe with Google module.
+ *
+ * @since 1.0.0
+ * @access private
+ * @ignore
+ */
+final class Subscribe_With_Google extends Module
+	implements Module_With_Assets, Module_With_Settings {
+	use Method_Proxy_Trait;
+	use Module_With_Assets_Trait;
+	use Module_With_Settings_Trait;
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since 1.0.0
+	 */
+	public function register() {
+		if ( ! $this->is_connected() ) {
+			return;
+		}
+
+		$is_amp   = $this->context->is_amp();
+		$settings = $this->get_settings()->get();
+
+		new EditPost( $settings );
+		new Header( $is_amp, $settings );
+		new ManagePosts();
+		new InlineEditPost( $settings );
+		new SinglePost( $is_amp );
+	}
+
+	/**
+	 * Returns all module information data for passing it to JavaScript.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return array Module information data.
+	 */
+	public function prepare_info_for_js() {
+		$info = parent::prepare_info_for_js();
+
+		$info['provides'] = array(
+			__( 'Generate revenue through your content by adding subscriptions or contributions to your publication', 'google-site-kit' ),
+		);
+
+		return $info;
+	}
```

Comment:
This override shouldn't be needed, the `provides` field isn't really relevant anymore.